core/model/modx/rest/modrest.class.php

```diff
@@ -473,7 +473,7 @@ public function toXml($parameters,$rootNode) {
      * @param array $parameters
      */
     protected function _populateXmlDoc(&$doc, &$node, &$parameters) {
-        /** @var $val DOMNode */
+        /** @var DOMnode/array $val */
```

Comment:
I don't think that's the right format for phpdoc is it? Isn't that supposed to be `DOMnode[]`?